.github/workflows/checklink.yaml

```diff
@@ -0,0 +1,14 @@
+name: Check Markdown links
+
+on: push
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: gaurav-nelson/github-action-markdown-link-check@v1
+      with:
+        use-quiet-mode: 'yes'
+        config-file: '.github/workflows/checklink_config.json'
+        max-depth: 3
```

Comment:
Why three?

Reply:
It is to ignore the directory of vendor.

Reply:
How about only check the `doc` and other necessary dir?

Reply:
How about only check the `doc` and other necessary dir?